### DISCUSSION TITLE
Welcome to README-Spotify-Status Discussions!

### DISCUSSION THREAD

**Discussion by sndrjhlncgr** [sndrjhlncgr/README-Spotify-Status#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bleyobeth** [comment]
### Hi man I am having a problem integrating spotify into README.md. Problem with Vercel service. I did everything as indicated in the instructions (README.md). I also tried to look for a solution to this problem on other sources, but from all of the above, I found only this video https://www.youtube.com/watch?v=n6d4KHSKqGk&t=555s. But I read the comments under the video, there people also write that they have a problem with the implementation of spotify in markdown. And when they build in Vercel, they get a 404 error: NOT_FOUND ###Code: NOT_FOUND.
### P.S. If you know how to fix this error, then write to me. Best regards, Kirill!
![image](https://user-images.githubusercontent.com/78875977/117589059-142d6d00-b130-11eb-917d-3fd0ae2721eb.png)

**Reply by sndrjhlncgr** [reply]
its about your vercel configuration i think.
